roles/installer/templates/deployments/web.yaml.j2

```diff
@@ -190,6 +186,47 @@ spec:
             {{ web_extra_env | indent(width=12, first=True) }}
 {% endif %}
           resources: {{ web_resource_requirements }}
+        - image: '{{ _image }}'
+          name: '{{ ansible_operator_meta.name }}-rsyslog'
+{% if rsyslog_command %}
+          command: {{ rsyslog_command }}
```

Comment:
will we ever reach here without `rsyslog_command` or `args` ? what happens in that case? we launch a container with the default command/arg called `myawx-rsyslog` ? I perhaps lost the context here, but these two conditionals, either should not exist or should condition the full existence of this container in the list and not just `command` or `args` fields imo